developer-setup.sh

```diff
@@ -20,7 +20,12 @@ echo "vscode ALL=(ALL) NOPASSWD:ALL" | sudo tee /etc/sudoers.d/vscode > /dev/nul
 
 echo "Installing other requirements now"
 
-curl -fsSL https://get.docker.com -o get-docker.sh && sudo sh get-docker.sh && sudo apt-get update && sudo apt install tmux jq figlet -y && sudo apt-get clean
+curl -fsSL https://get.docker.com -o get-docker.sh
+sudo sh get-docker.sh
+sudo apt-get update
+sudo apt install tmux jq figlet qemu-guest-agent -y
```

Comment:
Does this work when spinning up a VM adhoc. In the readme there is a break glass method to get up and running using a hetzner/EC2. Can you confirm this qemu-agent install doesn't break anything there?

Reply:
thanks for confirming